Dockerfile

```diff
@@ -135,6 +143,14 @@ RUN --mount=type=cache,target=/.cache cd /go/src/github.com/siderolabs/structpro
     && go build -o structprotogen . \
     && mv structprotogen /toolchain/go/bin/
 COPY --from=importvet /importvet /toolchain/go/bin/importvet
+COPY ./hack/ukify /go/src/github.com/siderolabs/ukify
+RUN --mount=type=cache,target=/.cache \
+    --mount=type=bind,source=pkg,target=/go/src/github.com/pkg \
+    cd /go/src/github.com/siderolabs/ukify \
+    && go build -o gen-certs ./gen-certs \
```

Comment:
nit: should we make the binary name more specific to the purpose, e.g. `gen-uki-certs` ?